doc/user/content/get-started/quickstart.md

```diff
@@ -447,66 +437,21 @@ Let's see this in action.
     (2 rows)
     ```
 
-1. Once the `r2` replica is ready (`ready=t`), drop the `r1` replica to simulate a failure:
-
-    ```sql
-    DROP CLUSTER REPLICA compute_qck.r1;
-    ```
-
-1. Query the indexed view that you created in `compute_qck` earlier:
-
-    ```sql
-    SELECT * FROM avg_bids;
-    ```
-    <p></p>
-
-    ```nofmt
-            item        |    average_bid
-    --------------------+--------------------
-     Custom Art         | 49.770776201263864
-     Gift Basket        |   49.8909070204407
-     City Bar Crawl     | 50.056635368698046
-     Best Pizza in Town |  50.50023551577956
-     Signed Memorabilia |  50.11854192264935
-    (5 rows)
-
-    Time: 23.537 ms
-    ```
-
-    As you can see, the results are available despite the failure of one of the cluster's replicas.
+    When the new replica shows `ready=t`, Materialize has acquired physical compute resources for the new replica and is catching it up to the initial replica. Once the new replica is caught up, the two replicas are logical clones that do the same computation and hold the same results.
 
 ## Step 6. Scale up or down
 
-In addition to using replicas to increase fault tolerance, you can add and remove replicas to scale resources up or down according to the needs of a cluster. For example, let's say the `2xsmall` replica in the `compute_qck` cluster is running low on memory.
+You can scale the physical compute resources of a cluster up or down based on need. For example, let's say `2xsmall` is not providing enough memory for the work of the `compute_qck` cluster.
 
-1. Add the next largest replica, `xsmall`:
+1. Use the [`ALTER CLUSTER`](/sql/alter-cluster/) command to bump the cluster up to the next largest size, `xsmall`:
 
     ```sql
-    CREATE CLUSTER REPLICA compute_qck.r3 SIZE = 'xsmall';
-    ```
-
-1. Use the [`SHOW CLUSTER REPLICAS`](https://materialize.com/docs/sql/show-cluster-replicas/) command to check the status of the new replica:
-
-    ```sql
-    SHOW CLUSTER REPLICAS WHERE CLUSTER = 'compute_qck';
+    ALTER CLUSTER compute_qck SET (SIZE 'xsmall');
     ```
-    <p></p>
 
-    ```noftm
-       cluster   | replica |  size   | ready
-    -------------+---------+---------+-------
-     compute_qck | r2      | 2xsmall | t
-     compute_qck | r3      | xsmall  | t
-    (2 rows)
-    ```
-
-1. Once the `r3` replica is ready (`ready=t`), it's safe to remove the `r2` replica:
-
-    ```sql
-    DROP CLUSTER REPLICA compute_qck.r2;
-    ```
+    Behind the scenes, this command adds a new `xsmall` replica and removes the `2xsmall` replica. There may be an interruption in the cluster's ability to serve queries while the new replica catches up to where the old replica left off.
```

Comment:
Maybe we should put the bit about interruption in a warning callout and suggest that if this is a problem, they can use an unmanaged cluster? @benesch, @antiguru

Reply:
This LGTM as is, FWIW. I think explaining unmanaged clusters is too in the weeds for the quickstart. Anyone that worried about uptime is likely in contact with the field engineering team in the short/medium term, and in the long term we'll have fixed this properly.